app/src/main/java/org/wikipedia/suggestededits/SuggestedEditsContributionsItemView.kt

```diff
@@ -0,0 +1,90 @@
+package org.wikipedia.suggestededits
+
+import android.content.Context
+import android.util.AttributeSet
+import android.view.View
+import android.view.ViewGroup
+import android.widget.LinearLayout
+import butterknife.OnClick
+import kotlinx.android.synthetic.main.item_suggested_edits_contributions.view.*
+import org.wikipedia.R
+import org.wikipedia.suggestededits.SuggestedEditsContributionsFragment.Contribution.Companion.EDIT_TYPE_IMAGE_CAPTION
+import org.wikipedia.suggestededits.SuggestedEditsContributionsFragment.Contribution.Companion.EDIT_TYPE_IMAGE_TAG
+import org.wikipedia.util.StringUtil
+import org.wikipedia.views.ViewUtil
+
+class SuggestedEditsContributionsItemView @JvmOverloads constructor(
+        context: Context,
+        attrs: AttributeSet? = null,
+        defStyle: Int = 0,
+        defStyleRes: Int = 0
+) : LinearLayout(context, attrs, defStyle, defStyleRes) {
```

Comment:
Looks like it does not need a `@JvmOverloads` annotation and can be simplified to:
```
SuggestedEditsContributionsItemView constructor(context: Context, attrs: AttributeSet? = null) : LinearLayout(context, attrs) {
```